meta/contao-themes-shop/vacancy-calendar/en.yml

```
@@ -0,0 +1,10 @@
+en:
+    title: Vacancy calendar
+    description: >
+        Allows the display of vacancies for different objects in a simple and convenient way. For each object a frontend-module is used.
```

```suggestion
        Allows the display of vacancies for different objects in a simple and convenient way. For each object a front end module is used.
```
The spell checkers complains about `frontend`. In English, `front end` is the correct term. Hence this change request.